DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -797,6 +798,38 @@ static function FixConversationAddNote(Conversation c, string textSnippet)
     }
 }
 
+function SetLampState(#var(prefix)Lamp l, bool turnOn) // could maybe go in DXRLamp if it existed
+{
+    if (turnOn && !l.bOn) {
```

Comment:
It's probably preferable to just always do this if turnOn, since the existing logic for what to do if a lamp is already on does not adjust ScaleGlow or bUnlit.